pkg/snapshotter/snapper-backend.go

```diff
@@ -49,60 +51,123 @@ type snapperBackend struct {
 func newSnapperBackend(cfg *types.Config, maxSnapshots int) *snapperBackend {
 	// snapper backend embeds an instance of a btrfs backend to fill the gap for the
 	// operatons that snapper can't entirely handle.
-	return &snapperBackend{cfg: cfg, maxSnapshots: maxSnapshots, btrfs: newBtrfsBackend(cfg, maxSnapshots)}
+
+	// TODO detect wheather the current snapper supports the required installation helper
+	return &snapperBackend{cfg: cfg, maxSnapshots: maxSnapshots, btrfs: nil}
```

Comment:
We should initiate btrfs backend if snapper does not support the latest installation helper and use it instead

Reply:
The PR is blocked until a new snapper version is released in TW. Then addressing this TODO will be needed.